components/script/dom/htmlelement.rs

```diff
@@ -1084,14 +1085,23 @@ impl VirtualMethods for HTMLElement {
         let element = self.as_element();
         match (attr.local_name(), mutation) {
             (name, AttributeMutation::Set(_)) if name.starts_with("on") => {
+                let doc = self.owner_document();
```

Comment:
Let's move this into EventTarget::set_event_handler_uncompiled, since we're missing https://github.com/servo/servo/blob/a1b9949f7563ae6ee30aedc0a3d0c86bb05d02fd/components/script/dom/htmlbodyelement.rs#L206 right now.